LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2020, xjamundx
```

Comment:
```suggestion
Copyright (c) 2016, xjamundx
```
Please review [this](https://github.com/xjamundx/eslint-plugin-promise#license). I believe it should be 2016, @xjamundx Correct me if I'm wrong and don't know if we should keep your `{{github_username}}` as the `{{ organization }}` or it's better to add your name instead i.e., `Jamund Ferguson`.

Reply:
I think it should be my name, yes, but I can probably do this.

Reply:
@xjamundx Awesome, thanks for the update.